CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 
 ### Removed
 - [Python] Drop compatibility for python 2. Supported python versions are 3.8, 3.9, 3.10, 3.12
+- [Python] Removed installation of `gherkin` script. It was used for internal acceptance tests only.
```

Comment:
BAD IDEA: Removing the scripts from the package description
* The user of this package wants to use the scripts and not some module statements that he/she does not know
* Using `gherkin/__main__.py` should be kept because that is the natural choice for the main program
* Usability for the user sucks if you do that
* It might be OK for the repo development in Makefile but even there, it is rather suboptimal (IMHO).

Reply:
actually, I'm even more inclined to not include the `scripts/` folder in the distribution at all.

The reason is that these scripts are only used internally, as there is absolutely no API or documentation about these. Hell, I can't even figure out what the `gherkin` script does at all, since there is no documentation.

I really doubt that any downstream user is using these. If anything, we are just polluting their PATH with this irrelevant `gherkin` executable.

In the remote hypothesis that a downstream user was relying on this, we can always release a hotfix with the script included in the distribution.

Reply:
If the executable is needed for the purposes of the user to understand how the public API works, wouldn't this be better handled with documentation?

From what I could see, the scripts seem to be necessary (or more necessary) when we had to deal with py2 and py3 and managing which command was invoked to run the tests.  As we only use py3, we don't need a script to do this for us now?

I may have missed something, but I didn't use this executable or use it to understand how to use gherkin when I implemented it in pytest-bdd.

Reply:
If these scripts/executables are only for internal use, then why put them in the Python package and not put the in `bin/` directory where they were before (at least one of them).

In addition, the `gherkin` script is provided in the current package version and before. Therefore, the script was part of the “API” of this package. By removing them you basically break this API in the next version. And you should be aware of that.

Reply:
> If these scripts/executables are only for internal use, then why put them in the Python package and not put the in bin/ directory where they were before (at least one of them

I can put them out of the `gherkin` package, so that they will automatically be out of the distributed package.

> In addition, the gherkin script is provided in the current package version and before. Therefore, the script was part of the “API” of this package. By removing them you basically break this API in the next version. And you should be aware of that.

yes I know, but every release is a major release it seems, so semver semantics allow this.

Reply:
> I can put them out of the gherkin package, so that they will automatically be out of the distributed package.

Done in b94911b060c7c1e15d4e200633a1341726cef7f6